solidity-checks/src/main/java/org/sonarsource/solidity/checks/CheckVerifier.java

```diff
@@ -56,13 +57,33 @@ public static void verifyNoIssue(IssuableVisitor check, String relativePath) {
     new CheckVerifier(check, relativePath, false);
   }
 
+  public static boolean verifyIssueOnFile(IssuableVisitor checkVisitor, String relativePath) {
+    File file = new File(relativePath); ///// TODO fix architecture
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=33&issues=AWRuvWkXI66nRh8YCYJl&open=AWRuvWkXI66nRh8YCYJl)

[ISSUE_KEY]: AWRuvWkXI66nRh8YCYJl